doc/source/tune/api_docs/stoppers.rst

```diff
@@ -0,0 +1,55 @@
+.. _tune-stoppers:
+
+Stopping mechanisms (tune.stopper)
+==================================
+
+Apart from scheduling decisions like :ref:`ASHA <tune-scheduler-hyperband>`, where a number of
+trials are stopped if they perform subpar, Ray Tune offers abstractions to implement own stopping
+mechanisms. For instance, you can specify to stop trials when they reached a plateau and the metric
+doesn't change anymore.
+
+Ray Tune comes with several stopping mechanisms out of the box. For custom stopping behavior, you can
+inherit from the :class:`Stopper <ray.tune.Stopper>` class.
+
+.. contents::
+    :local:
+    :depth: 1
+
+
+.. _tune-stop-ref:
+
+Stopper (tune.Stopper)
+----------------------
+
+.. autoclass:: ray.tune.Stopper
+    :members: __call__, stop_all
+
+CombinedStopper (tune.stopper.CombinedStopper)
+----------------------------------------------
+
+.. autoclass:: ray.tune.stopper.CombinedStopper
```

Comment:
should we actually expose this? 

I'm thinking we can just support passing in a list of stoppers instead of having the user compose it themselves.

Reply:
For reference, in Keras, the "Combiner Callback" class is not exposed to the user.

Reply:
That makes sense. I'll add the functionality to `tune.run()`.